src/carplay.cs

```diff
@@ -686,9 +699,12 @@ interface ICPSessionConfigurationDelegate { }
 	[BaseType (typeof (NSObject))]
 	interface CPSessionConfigurationDelegate {
 
-		[Abstract]
```

Comment:
Add `[Abstract]` again, removing it causes a breaking change

Reply:
If it's not `@required` you can add a comment like `// this is not @required since Xcode 11 but kept for API compatibility`

Reply:
and, if that's the case, you'll likely need to update `iOS-CarPlay.ignore` too :)

Reply:
updated .cs and .ignore! :)